bot/cogs/snakes.py

```diff
@@ -28,8 +75,44 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         :return: A dict containing information on a snake
         """
+        base_url = "https://protected-reef-75100.herokuapp.com/"
```

Comment:
Huh, that's an odd decision.

Reply:
Can we get more info on what you're doing here?

Reply:
Hey @gdude2002 , that's where we hosted our snake API. Here's the [link](https://github.com/prithajnath/snake) to the repo. This is where most of my commits are :)